app/controllers/application_controller.rb

```diff
@@ -24,7 +24,6 @@ class ApplicationController < ActionController::Base
   before_action :redirect_to_proper_context!, if: :immersive_context_wrong?
 
   before_action :authorize_if_private!
-  before_action :validate_active_organization!
```

Comment:
I didn't delete this action, it's duplicated, that's why I remove one of both... See line 22 of this file